whenchanged/whenchanged.py

```diff
@@ -13,6 +13,14 @@
 -1 Don't re-run command if files changed while command was running
 -s Run command immediately at start
 
+Environment variables:
+- WHEN_CHANGED_EVENT: reflects the current event type that occures.
+    Could be either:
+        - file_created
```

Comment:
Please compress this list to one line to avoid a very long help message.

Reply:
Done